api/http/tests/test_indigo_http.py

```diff
@@ -1,10 +1,15 @@
+import base64
+import io
 import itertools
 import os
 import pathlib
-from typing import Any, List
+import xml.etree.ElementTree as elTree
+from typing import Any, Dict, List
 
+import PyPDF2
 import pytest
 from fastapi.testclient import TestClient
+from PIL import Image
```

Comment:
We are using new dependencies here, like `Pillow` or `PyPDF2`, so please add them to `api/http/requirements_dev.txt`.

Reply:
I noticed a couple of other dependencies and will add them too.